tests/e2e/monitoring/test_mon_log_trimming.py

```diff
@@ -0,0 +1,171 @@
+import logging
+import pytest
+import random
+import threading
+import re
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import E2ETest, bugzilla
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.resources.pod import get_mon_pods
+
+log = logging.getLogger(__name__)
+
+
+@bugzilla("1941939")
+@pytest.mark.polarion_id("OCS-2526")
+class TestMonLogTrimming(E2ETest):
+    """
+    Test cases to validate mon store trimming
+
+    Steps:
+      - Change values:
+        paxos_service_trim_min default value = 250
+        paxos_service_trim_max default value = 500
+        osd_op_complaint_time default value = 30.000000
+      - Execute ops that will write to mon store db
+      - Sample the db size and check that it actually get trimmed
+      - Change the above to default values
+    """
+
+    selected_mon_pod = None
+    ct_pod = None
+    initial_db_size = 0
+    mon_db_trim_count = 0
+    current_mon_db_size = 0
+    stop_checking_mon_db = False
+    fio_pod_obj = None
+    MAX_DELTA_DB_SIZE = 0
+    PAXOS_MIN_DEFAULT = 250
+    PAXOS_MAX_DEFAULT = 500
+    PAXOS_COMPLAINT_TIME_DEFAULT = 30.000000
+
+    @pytest.fixture(autouse=True)
+    def setup(self, request, pod_factory):
+        """
+        Set values for:
+          paxos_service_trim_min=10
+          paxos_service_trim_max=100
+          osd_op_complaint_time=0.000001
+        """
+        self.fio_pod_obj = pod_factory(constants.CEPHFILESYSTEM)
```

Comment:
What is the size of PVC that it will create here?

Reply:
Since we're changing the default values for paxos_service_trim_min, paxos_service_trim_max and osd_op_complaint_time
The trimming will be triggered every few MiB so IMO it doesn't really matter.